src/app/actions.ts

```diff
@@ -62,3 +63,28 @@ const speciesFuseIndex = new Fuse<Species>(speciesData as readonly Species[], {
 export async function findSpecies(query: string): Promise<SpeciesGetResponse> {
   return speciesFuseIndex.search(query).slice(0, 10);
 }
+
+export async function getNearbySightings(longitude: number, latitude: number) {
```

Comment:
This function is here so it can run on the server. It was originally with the map.tsx file where it is called.